tests/utils/test_agent_display_console.py

```
@@ -0,0 +1,175 @@
+import pytest
+import asyncio
```

Ruff (0.11.9)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)